src/vi/mangaxy/src/eu/kanade/tachiyomi/extension/vi/MangaXY/MangaXY.kt

```diff
@@ -0,0 +1,387 @@
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.ParsedHttpSource
+import okhttp3.Headers
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.OkHttpClient
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import rx.Observable
+import java.text.SimpleDateFormat
+import java.util.Locale
+import java.util.TimeZone
+
+class MangaXY : ParsedHttpSource() {
+
+    override val name = "MangaXY"
+
+    override val baseUrl = "https://mangaxy.com"
+
+    override val lang = "vi"
+
+    override val supportsLatest = true
+
+    override val client: OkHttpClient = network.cloudflareClient
+
+    override fun headersBuilder() = Headers.Builder()
+        .add("Referer", "$baseUrl/")
+
+    override fun popularMangaRequest(page: Int) = GET(
+        baseUrl.toHttpUrl().newBuilder().apply {
+            addPathSegment("search.php")
+            addQueryParameter("act", "search")
+            addQueryParameter("sort", "xem")
+            addQueryParameter("view", "thumb")
+            addQueryParameter("page", page.toString())
+        }.build().toString(),
+        headers
+    )
+    override fun popularMangaSelector() = ".container > .row > div.col-12.col-lg-9 > #tblChap > .thumb"
+
+    override fun latestUpdatesSelector() = popularMangaSelector()
+
+    override fun latestUpdatesRequest(page: Int) = GET(
+        baseUrl.toHttpUrl().newBuilder().apply {
+            addPathSegment("search.php")
+            addQueryParameter("act", "search")
+            addQueryParameter("sort", "chap")
+            addQueryParameter("view", "thumb")
+            addQueryParameter("page", page.toString())
+        }.build().toString(),
+        headers
+    )
+
+    override fun popularMangaFromElement(element: Element): SManga {
+        val manga = SManga.create()
+        element.select("a.name").first().let {
+            manga.setUrlWithoutDomain(it.attr("href"))
+            manga.title = it.text().trim()
+        }
+        manga.thumbnail_url = element.select(".item img")
+            .first()
+            .attr("style")
+            .substringAfter("url('")
+            .substringBefore("')")
+            .replace("//", "https:")
+            .replace("http:", "")
+        return manga
+    }
+
+    override fun latestUpdatesFromElement(element: Element): SManga {
+        val manga = SManga.create()
+
+        element.select("a.name").first().let {
+            manga.setUrlWithoutDomain(it.attr("href"))
+            manga.title = it.text()
+        }
+
+        manga.thumbnail_url = element.select(".item img")
+            .first()
+            .attr("style")
+            .substringAfter("url('")
+            .substringBefore("')")
+            .replace("//", "https:")
+            .replace("http:", "")
+        return manga
+    }
```

Comment:
```suggestion
    override fun latestUpdatesFromElement(element: Element): SManga = popularMangaFromElement(element)
```
they are the same so why not